supervisor/homeassistant/module.py

```diff
@@ -60,12 +60,15 @@
 
 
 HOMEASSISTANT_BACKUP_EXCLUDE = [
+    "**/__pycache__/*",
+    "**/.DS_Store",
     "*.db-shm",
     "*.corrupt.*",
-    "__pycache__/*",
-    "*.log",
     "*.log.*",
+    "*.log",
     "OZW_Log.txt",
+    "backups/*.tar",
+    "tmp_backups/*.tar",
```

Comment:
Technically those two are really only required for Core installations, since in Supervisor we store backups elsewhere. However, they don't really hurt. And maybe if someone just copied the `/config` directory, it might even be helpful :man_shrugging: